src/Chat/Message.php

```diff
@@ -9,4 +9,40 @@ final class Message
     public ChatRole $role;
 
     public string $content;
+
+    public static function system(string $content): self
+    {
+        $message = new self();
+        $message->role = ChatRole::System;
+        $message->content = $content;
+
+        return $message;
+    }
+
+    public static function user(string $content): self
+    {
+        $message = new self();
+        $message->role = ChatRole::User;
+        $message->content = $content;
+
+        return $message;
+    }
+
+    public static function assistant(string $content): self
+    {
+        $message = new self();
+        $message->role = ChatRole::Assistant;
+        $message->content = $content;
+
+        return $message;
+    }
+
+    public static function function(string $content): self
```

Comment:
what would be the usage of the function "function" ? and maybe not use the word function in case it breaks some tools.

Reply:
renamed it to `functionCall` :)